### DISCUSSION TITLE
Stille Karten + Beschriftungslayer + Strassennetz

### DISCUSSION THREAD

**Discussion by loni13** [GeoUnconference/discussions#25] [Ideen / Idées]
Für verschiedene Projekte wären für uns stille Karten (ohne Beschriftungen) + Beschriftungslayer (z.B. mit Ortsnamen, Strassennamen, Parzellen) wünschenswert. Damit könnte man sehr einfach identische Kartenausschnitte mit der gleichen Beschriftung, aber anderer Basiskarte erstellen und nebeneinander abbilden.

Im Idealfall kombinierbar mit einem Strassennetz (als (Poly-)Linienzüge mit Eigenschaften wie erlaubter Fahrtrichtung, Strassentyp, Breite, etc.). Optimal wäre, wenn man für die Strassen dann auch noch ein Höhenprofil hätte.

Wir arbeiten für statische Pläne mit ArcGIS und entwickeln auch Webapplikationen mit dynamischen Daten.

**Comment by davidoesch** [comment]
Stumme Karten:

**A) https://vectortiles.geo.admin.ch mit offiziellen swisstopo Daten:**
Leichte Basiskarte https://www.swisstopo.admin.ch/de/geodata/maps/smw/smw_lightbase.html oder Imagery Base Map https://www.swisstopo.admin.ch/de/geodata/maps/smw/smw_imagerybase.html 

Sie können die Darstellung eines der  bestehenden Vector Tile Datensatzes selbst nach Ihren Wünschen und Bedürfnissen ändern und diese auf einer von ihnen genutzen  Platfform im Internet zugänglich zu machen. Bitte beachten Sie, dass nur Mitgliedern der Bundesverwaltung eine weltweite Abdeckung mit OpenMapTiles-Daten für selbst gehostete Kartendesigns zur Verfügung steht. Die Schweiz und Lichtenstein sind frei verfügbar und das entsprechende Style frei  verfügbar. 

1. Öffnen Sie die entsprechende style Datei (JSON-Datei) https://api3.geo.admin.ch/services/sdiservices.html#getstyle  mit einem webbasierten Vektor-Style-Editor z.B., maputnik Editor https://maputnik.github.io/editor/  oder maptiler Editor https://cloud.maptiler.com/
2. Bearbeiten Sie die Karte nach Ihren Bedürfnissen
3. Exportieren sie ihre individualiserten style als  JSON-Datei. und hosten  sie diese Datei auf einem Server, auf den über eine URL zugegriffen werden kann.

Nutzung ihres individualisierten styles:

1. Darstellung ihres styles in test.map.geo.admin.ch:  Sie müssen die URL des Viewers wie folgt anpassen: Hinzufügen des Parameters "&bgLayer_styleUrl=" gefolgt von der URL zu Ihrer style Datei, z.B. https://test.map.geo.admin.ch/?lang=de&bgLayer_styleUrl=https://cms.geo.admin.ch/www.geo.admin.ch/cms_api/gl-styles/lbm_v1.0.0_simplified_grey.json
2. Oder in einer eigenen Anwendung.

**B) RASTER  WMTS /WMS mit offiziellen swisstopo Daten:**
Es gibt verschiedene offizielle Rasterdatensätze, die sich zu Karten kombinieren lassen



- Relief ch.swisstopo.swissalti3d-reliefschattierung https://s.geo.admin.ch/92eccc918e
- Relief ch.swisstopo.leichte-basiskarte_reliefschattierung https://s.geo.admin.ch/92eccdd649 
- Relief ch.bafu.landesforstinventar-vegetationshoehenmodell_relief https://s.geo.admin.ch/92ecce78a7 
- Relief ch.bafu.gefahren-basiskarte https://s.geo.admin.ch/92eccf96c5
-  SwissTLM: Stumm in der übersicht: ch.swisstopo.swisstlm3d-karte-farbe https://s.geo.admin.ch/7ec8198bbc und aufgehellt https://s.geo.admin.ch/7ec819e73e  
- Strassennamen OSM + WMTS swisstopo: https://s.geo.admin.ch/92ecc48cbb 
-  Strassennamen swisstopo + WMTS swisstopo: https://s.geo.admin.ch/92ecd0f0a0 

und diese dann in Kombinationen mit einer der 850 layer...
- Grau in Übersicht: ch.bafu.hydrologie-hintergrundkarte https://s.geo.admin.ch/92ecc95dc6
- Custom Hintergrund FARBIG/ Grau:  https://s.geo.admin.ch/6aaaf4666e
- Custom Hintergrund FARBIG ÜBERSICHT: https://s.geo.admin.ch/6a72bb7a50
- Custom Hintergrund FARBIG Detail: https://s.geo.admin.ch/6a72b6e71b 
- Custom Hintergrund FARBIG Detail auch in 3D https://s.geo.admin.ch/6a72b95e71

**erlaubter Fahrtrichtung, Breite**
Diese Daten gibt es nicht offiziell bei swisstopo Daten (ausser Strassenklasse), aber in den AV Daten, haben geometrien bzgl Breite drin zb via https://s.geo.admin.ch/92ecd2a8f9 die Rohdaten können via https://data.geo.admin.ch/ch.swisstopo-vd.amtliche-vermessung/data.zip bezogen werden

**Strassen dann auch noch ein Höhenprofil hätte.**

Das kann für jegwelche Geometrie vons wisstopo Elementen via https://www.swisstopo.admin.ch/de/geodata/landscape/tlm3d.html#download bezogen werden oder via Geodienst profile https://api3.geo.admin.ch/services/sdiservices.html#profile  direkt abgefragt werden

**Reply by rastrau** [reply]
Allenfalls lohnt es sich, beim Strassennetz einen Bezug auch zu https://github.com/GeoUnconference/discussions/discussions/10 zu machen bzw. sich dazu mit @datamapio an der 2. GeoUnconference von diesem Donnerstag auszutauschen.

**Reply by loni13** [reply]
@davidoesch 
Vielen Dank für die ausführliche Antwort.

Wir arbeiten im Normalfall mit SwissTLM, Pixelmap, Cadastral und swissimage und sind auf aktuelle Daten angewiesen. Von daher sind selbst gehostete Karten nicht ganz so optimal für uns.

Die SwissTLM-Karten zeigen z.B. in Thun Strassennamen und sind wohl nicht überall stumm (?)
Gibt es die Strassennamen auch als WMTS-Dienst? 

Die Geodaten für die Höhenangaben waren mir bekannt, aber nicht der Geodienst. Das ist ein toller Hinweis, den ich sehr gerne in unsere Applikation einbauen werde.

@rastrau 
Ja - da ist ein Austausch sicher interessant. Ich bin am Donnerstag vor Ort.

**Reply by davidoesch** [reply]
Die SwissTLM-Karten zeigen z.B. in Thun Strassennamen und sind wohl nicht überall stumm (?)
-> korrekt: dazu am besten die Vectortiles nutzen: da können gemäss Amleitung Elemente ausgeblendet werden

Gibt es die Strassennamen auch als WMTS-Dienst
-> da täglich upgedated nur als WMS

**Comment by schenardi** [comment]
Zur Originalfrage folgendes:

* Die Vectortiles von Swisstopo sind bekanntlich leider in Esri Clients standardmässig nicht nutzbar (vergleiche auch Diskussion aus Unconf1 #8 )
* Esri Schweiz hat auf ArcGIS Online Vector Tile Layers publiziert, z .B. in LV95: https://esrich-content.maps.arcgis.com/home/item.html?id=2345743b15e04215aac3845d67b556da oder auch in WGS84
* Mit dem Vector Tile Style Editor können beliebige Styles ergänzt werden, z.B. ohne Beschriftungen. Die Basisdaten verbleiben dieselben wie im Dienst. https://developers.arcgis.com/vector-tile-style-editor/

**Reply by davidoesch** [reply]
Top—- zu ergänzen: der ESRI Dienst beruht in der Schweiz auf den offiziellen swisstopo Daten . Same content as in vectortiles
Geo.admin.ch

**Reply by loni13** [reply]
Vielen Dank für die Ergänzung.

Ich lese dort: "Important Note: This item is in beta and is not intended for use in production applications." 
Gibt es Informationen zur Weiterentwicklung / -pflege?

**Reply by schenardi** [reply]
Diese Frage müsste Esri CH beantworten, ev. ist die M.R. von Esri CH morgen bei der Konferenz auch wieder vertreten?

**Reply by rastrau** [reply]
Ja, die angesprochene Vertreterin von Esri ist angemeldet 👍

**Reply by mrueegsegger** [reply]
@loni13 Besser spät als nie:
Die angesprochenen Vector Tile Basemaps in LV95, z.B. [LV95 Swiss Topographic Map](https://esrich-content.maps.arcgis.com/home/item.html?id=48dd74b895c249dd94ffa67c835961ba) sind nun nicht mehr in Beta und können produktiv eingesetzt werden. Als Ergänzung: In allen ArcGIS Vector Tile Basemaps (WebMercator, WGS84) werden die swissTLM3D-Daten verwendet.
